### ISSUE TITLE
Enable cors for all endpoints

### ISSUE THREAD

**Issue by simonhoss** [VictoriaMetrics/VictoriaMetrics#5623]
### Is your feature request related to a problem? Please describe

I want to query all endpoints with cors enabled, at the moment not every endpoint has this enabled. For example the /export does not have cors enabled.

### Describe the solution you'd like

Enable cors for all endpoints.

**Comment by dandv** [comment]
Good idea. Any other endpoints besides [export](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/10899) that don't have CORS enabled?
